core/domain/state_domain_test.py

```diff
@@ -4433,6 +4433,55 @@ def test_traverse_by_schema_and_get(self):
         self.assertEqual(html, ['<p>testing</p>'])
 
 
+class InteractionInstanceDomainTests(test_utils.GenericTestBase):
+
+    def test_to_proto(self):
+        subtitled_html = state_domain.SubtitledHtml(
+            'default_outcome', '')
+        default_outcome = state_domain.Outcome(
+            'end_state_name', subtitled_html, False, [], None, None)
+        customization_args = (
+            state_domain.InteractionInstance.convert_customization_args_dict_to_customization_args( # pylint: disable=line-too-long
+                'Continue',
+                {
+                    'buttonText': {
+                        'value': {
+                            'content_id': 'ca_buttonText_1',
+                            'unicode_str': 'Click Me!'
+                        }
+                    }
+                }
+            ))
+        continue_interaction = state_domain.InteractionInstance(
+            'Continue',
+            customization_args,
+            [], default_outcome, [], [], None)
+        continue_interaction_proto = (
+            continue_interaction.to_proto())
+        self.assertEqual(
+            continue_interaction_proto.continue_instance.customization_args.button_text.text, # pylint: disable=line-too-long
```

Comment:
A better way to handle this is to declare a variable for continue_interaction_proto.continue_instance.

Then you can use that variable in these three asserts, and avoid the "line too long" issue.

Reply:
Done